Makefile

```diff
@@ -612,27 +621,36 @@ $(HELM):
 $(HOME)/.helm:
 	$(HELM) init --client-only
 
+.PHONY: helm-repo-add
+
+helm-repo-add:
+	$(HELM) repo add istio.io ${ISTIO_HELM_REPO}
+
 # create istio-remote.yaml
-istio-remote.yaml: $(HELM) $(HOME)/.helm
+istio-remote.yaml: $(HELM) $(HOME)/.helm helm-repo-add
 	cat install/kubernetes/namespace.yaml > install/kubernetes/$@
 	$(HELM) dep update --skip-refresh install/kubernetes/helm/istio-remote
 	$(HELM) template --name=istio --namespace=istio-system \
+		--set istio_cni.enabled=${ENABLE_ISTIO_CNI} \
+		${EXTRA_HELM_SETTINGS} \
 		install/kubernetes/helm/istio-remote >> install/kubernetes/$@
 
 # creates istio.yaml istio-auth.yaml istio-one-namespace.yaml istio-one-namespace-auth.yaml
 # Ensure that values-$filename is present in install/kubernetes/helm/istio
-isti%.yaml: $(HELM) $(HOME)/.helm
+isti%.yaml: $(HELM) $(HOME)/.helm helm-repo-add
```

Comment:
you will end up with 8-10 helm repo adds here (one per values.yaml file.  Is that intentional?

Reply:
The intent was to get the repo-add prior to any dep update.  Repo-add seems idempotent (multiple runs are ok)